frontend/app-config/webpack.config.js

```diff
@@ -329,7 +329,10 @@ module.exports = function (env) {
                 entryModule: 'app/app.module#AppModule',
                 skipCodeGeneration: !isAot,
                 sourceMap: !isProduction,
-                tsConfigPath: configFile
+                tsConfigPath: configFile,
```

Comment:
What is all this? We do not use angular i18n

Reply:
Removed, I still had to clean it, my bad